Schema/Schema.php

```diff
@@ -18,55 +18,48 @@ class Schema implements \ArrayAccess
      * @var string
      *
      * @JMS\Type("string")
-     * @JMS\Exclude(if="object.ref !== null")
```

Comment:
It was introduced in one of the latest master-commits, but unfortunately broke everything for me. That's why i removed it.